lighthouse-core/test/lib/i18n/i18n-test.js

```diff
@@ -76,4 +76,47 @@ describe('i18n', () => {
       expect(i18n.lookupLocale('jk-Latn-DE-1996-a-ext-x-phonebk-i-klingon')).toEqual('en');
     });
   });
+
+  describe('#_formatIcuMessage', () => {
```

Comment:
oh yeah, maybe rename the `describe`. `describe('Message argument formatting', () => {` or something?